lib/vk_format.h

```diff
@@ -1207,6 +1208,19 @@ static inline void vkGetFormatSize( const VkFormat format, ktxFormatSize * pForm
 			pFormatSize->blockHeight = 4;
 			pFormatSize->blockDepth = 1;
 			break;
+		case VK_FORMAT_PVRTC1_2BPP_SRGB_BLOCK_IMG:
+		case VK_FORMAT_PVRTC1_2BPP_UNORM_BLOCK_IMG:
+        case VK_FORMAT_PVRTC2_2BPP_SRGB_BLOCK_IMG:
+        case VK_FORMAT_PVRTC2_2BPP_UNORM_BLOCK_IMG:
```

Comment:
Should these cases be split as well (also 4bpp options below)?

Reply:
Why? What do you mean?

Reply:
PVRTC2 shouldn't get `minBlocksX = 2`.

Reply:
Fixed.